src/coreclr/scripts/superpmi.py

```diff
@@ -1971,17 +1972,19 @@ def replay_with_asm_diffs(self):
 
         # These vars are force overridden in the SPMI runs for both the base and diff, always.
         replay_vars = {
-            "DOTNET_JitAlignLoops": "0", # disable loop alignment to filter noise
             "DOTNET_JitEnableNoWayAssert": "1",
             "DOTNET_JitNoForceFallback": "1",
         }
 
+        if not self.coreclr_args.alignloops:
+            replay_vars.update({
+                "DOTNET_JitAlignLoops": "0" })
+
         # These vars are used in addition to the replay vars when creating disassembly for the interesting contexts
         asm_dotnet_vars = {
             "DOTNET_JitDisasm": "*",
             "DOTNET_JitUnwindDump": "*",
             "DOTNET_JitEHDump": "*",
-            "DOTNET_JitDiffableDasm": "1", # to be removed
```

Comment:
remember to undo this.

Reply:
Why? This is the obsolete switch which has been replaced with JitDisasmDiffable, which is still set.

Reply:
I misread your comment about "# to be removed" thinking that you want to revert this change before merging.